frontend/tests/features/collection/collection.test.ts

```diff
@@ -32,6 +32,16 @@ type CollectionFormInput = Pick<
 >;
 
 describe("Collection", () => {
+  describe("Unit tests", () => {
```

Comment:
move unit test into its own `describe` block, so we can mark the logged in tests with tag `@loggedIn`